packages/core/src/providers/ping-id.ts

```diff
@@ -0,0 +1,69 @@
+import type { OAuthConfig, OAuthUserConfig } from "./index.js"
```

Comment:
Could you take the Auth0 provider as an example regarding JSDoc format and add doc for this one 🙏 https://github.com/nextauthjs/next-auth/blob/main/packages/core/src/providers/auth0.ts